testonly/hammer/hammer.go

```diff
@@ -671,24 +671,17 @@ leafloop:
 		}
 	}
 
-	root, err := s.vc.SetAndVerifyMapLeaves(ctx, leaves, metadataForRev(uint64(rev+1)))
+	writeRev := uint64(rev + 1)
+	_, err := s.vc.SetAndVerifyMapLeaves(ctx, leaves, metadataForRev(writeRev))
 	if err != nil {
 		return fmt.Errorf("failed to set-leaves(count=%d): %v", len(leaves), err)
 	}
 
-	s.pushSMR(root)
-	newContents, err := s.prevContents.UpdateContentsWith(root.Revision, leaves)
+	_, err = s.prevContents.UpdateContentsWith(writeRev, leaves)
 	if err != nil {
 		return err
 	}
-	wantRootHash, err := newContents.RootHash(s.cfg.MapID, s.vc.Hasher)
-	if err != nil {
-		return err
-	}
-	if !bytes.Equal(root.RootHash, wantRootHash) {
-		return fmt.Errorf("failed to check root hash: got %x, want %x", root.RootHash, wantRootHash)
-	}
-	glog.V(2).Infof("%d: set %d leaves, new SMR(time=%q, rev=%d)", s.cfg.MapID, len(leaves), time.Unix(0, int64(root.TimestampNanos)), root.Revision)
+	glog.V(2).Infof("%d: set %d leaves, rev=%d", s.cfg.MapID, len(leaves), rev)
```

Comment:
Should this print `writeRev` instead?

Reply:
This method inside the hammer is still called setLeaves so I decided to leave it as-is. In a future life I will come back and rename everything in the hammer, but it would be better to be a standalone PR as it will cause a large line delta.

Reply:
I mean, previously this line printed `root.Revision` which corresponded to `writeRev`, not `rev`.

Reply:
Good call. Thanks for clarifying,